apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```
@@ -149,7 +149,7 @@ div.pl-order-blocks-header {
   margin-bottom: 0px !important;
 }
 
-.pl-order-blocks-selected:focus {
-  outline: 3px solid #ff6200;
-  outline-offset: -3px;
+[aria-selected='true']:not(.ui-sortable-placeholder) {
+  outline: 2px solid orangered;
+  outline-offset: -2px;
 }
```

This rule as written will apply to objects that are not related to this element. This should have some distinction to ensure that it doesn't affect other elements. Something like (please test):
```suggestion
.pl-order-blocks-container [aria-selected='true']:not(.ui-sortable-placeholder) {
  outline: 2px solid orangered;
  outline-offset: -2px;
}
```